packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -116,18 +117,16 @@ const multiChainFetchList: ChainPair[] = [
     chain: ChainId.ArbitrumSepolia
   },
   // Orbit
-  {
-    parentChain: ChainId.ArbitrumOne,
-    chain: ChainId.Xai
-  },
-  {
-    parentChain: ChainId.ArbitrumGoerli,
-    chain: ChainId.XaiTestnet
-  },
   {
     parentChain: ChainId.ArbitrumSepolia,
     chain: ChainId.StylusTestnet
   },
+  ...Object.values(orbitChains).map(orbitChain => {
```

Comment:
I see we do `Object.values(orbitChains)` in a few places, so let's export a `getOrbitChains` function that does it and call that instead

Reply:
done